connector_dicom/models/res_partner.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (C) 2020 Konos
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+from datetime import datetime
+import requests
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+
+    dicom_score = fields.Char(
+        compute="_get_dicom", inverse="_set_dicom", string="Dicom Score")
+    dicom_date = fields.Date(
+        compute="_get_dicom", inverse="_set_dicom", string="Dicom Date")
+    dicom_count = fields.Integer(
+        compute="_get_dicom_count", string="Dicom Count")
+    dicom_ids = fields.Many2many(
+        "partner.dicom", string='Dicoms', compute="_get_dicom_count",
+        readonly=True, copy=False)
+
+    def _get_dicom(self):
+        dicom = self.env['partner.dicom']
+        for record in self:
+            partner_dicom = dicom.search([('partner', '=', record.id)],
+                                         limit=1, order='date desc')
+            if partner_dicom:
+                record.dicom_date = partner_dicom.date
+                record.dicom_score = partner_dicom.dicom_score
+                if partner_dicom.dicom_score < "999":
+                    record.dicom_ok = True
+                else:
+                    record.dicom_ok = False
+
+    def _set_dicom(self):
+        for record in self:
+            if record.dicom_score and record.dicom_date:
+                if not self._dicom_exist():
+                    data = {
+                    'dicom_score': record.dicom_score,
+                    'creation_date': fields.Date.context_today(record),
+                    'date': self.dicom_date,
+                    'partner': record.id}
+                    self.env['partner.dicom'].create(data)
+            else:
+                _logger.warning("Fecha")
+                _logger.warning(self.dicom_date)
+                _logger.warning(record.dicom_score)
+
+    def _dicom_exist(self):
+        dicom_model = self.env['partner.dicom']
+        for record in self:
+            dicom_found = dicom_model.search([
+                ('partner','=', record.id),
+                ('date','=', record.dicom_date)])
+            if dicom_found:
+                return True
+            else:
+                return False
```

Comment:
just `return dicom_found`, or `return bool(dicom_found)` if you really insist on an returning a boolean.
But returning the record would make it easy to update it, per my previous comment.